apps/lido/arapp.json

```diff
@@ -15,6 +30,11 @@
       "id": "MANAGE_WITHDRAWAL_KEY",
       "params": []
     },
+    {
+      "name": "Manage protocol contracts role",
+      "id": "MANAGE_PROTOCOL_CONTRACTS_ROLE",
```

Comment:
Maybe we should remove the obsolete roles?
* `SET_ORACLE`
* `SET_TREASURY`
* `SET_INSURANCE_FUND`